data/json/items/armor/torso_armor.json

```diff
@@ -225,6 +225,62 @@
       }
     ]
   },
+  {
+    "id": "chestguard_metal_sheets",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "sheet metal chest guard" },
+    "description": "Two sheets of metal hammered to fit your torso and riveted to leather belts to act as straps.  Heavy, uncomfortable, and difficult to move in, but it should help keep your blood inside your body.",
+    "weight": "8800 g",
+    "//": "The weight assumes the sheet metal is 75cm x 50cm x 1.5mm.",
+    "volume": "4 L",
+    "price": 500,
+    "price_postapoc": 400,
+    "to_hit": -2,
+    "material": [ "steel" ],
+    "symbol": "H",
+    "looks_like": "cuirass_lightplate",
+    "color": "light_gray",
+    "warmth": 10,
```

Comment:
I'm pretty sure 1.5mm of *metal* has near-zero insulation value to it 🤔

(P.S. I think all of the existing pure-metal armors generally have too high warmth)

Reply:
Yeah, technically they'd have negative insulation if that was possible, but I don't really think there's a way to make armor that sucks to wear in both winter and summer. I guess I could make it 0 warmth like chainmail and just let the 0 breathability represent that it'd still suck in summer.

Reply:
Other pure-metal armors do have token warmth values though, and 10 is fairly low (lower than what they were previously at 20 warmth for the arm guards), so on review I'll probably just keep the warmths as-is. I'd love to fix it up if we ever get a proper insulation system though (though in that case it'll probably be handled automatically anyway).